<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neato xv11 viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <widget class="serialClass" name="widgetSerial" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="draw360" name="widgetDrawer" native="true"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Tools">
       <item>
        <widget class="QGroupBox" name="groupBoxMode">
         <property name="title">
          <string>Mode</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="radioButton_Mode_Off">
            <property name="text">
             <string>Off</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_Mode_On">
            <property name="text">
             <string>On</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_Mode_Spin">
            <property name="text">
             <string>Spin</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>RPM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxSpeed">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderSpeed">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditSpeed">
         <property name="text">
          <string>0.00 RPM</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>draw360</class>
   <extends>QWidget</extends>
   <header>draw360.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>serialClass</class>
   <extends>QWidget</extends>
   <header>serialclass.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
